lib/daru/index/multi_index.rb

```diff
@@ -171,8 +185,20 @@ def find_all_indexes label, level_index, chosen
       end
     end
 
+    # Array `name` must have same length as levels and labels.
+    def validate_name names, levels
+      error_msg = 'names and levels should be of same size. size of the name '\
+      'array is #{names.size} and size of the MultiIndex levels and labels is'\
+      ' #{labels.size}.'
```

Comment:
Are you sure it will produce reasonable output? Singular quotes do not interpolate `#{}`, so you'll end up with message saying exactly `array is #{names.size}`, instead of proper number.